migrations/20240321211311_event.up.sql

```diff
@@ -0,0 +1,18 @@
+-- Add up migration script here
+
+CREATE TABLE IF NOT EXISTS event (
+    id BLOB NOT NULL, -- network_id sort_value controller StreamID height event_cid
+    cid BLOB UNIQUE, -- NOT NULL? the cid of the event as bytes no 0x00 prefix
```

Comment:
do we want to store event cid in this table or just use `select cid from event_block where id = $1`

should this be NOT NULL?

Reply:
decided to use CID as the event PK that other tables reference and order_key as a unique index for recon splits